### ISSUE TITLE
Configuration is broken after renew

### ISSUE THREAD

**Issue by pingu8007** [Neilpang/acme.sh#1568]
Steps to reproduce
------------------
* Start dockerized acme.sh daemon and upgrade.
* Issue new cert for `example.com` with `ec-256` private key, `dns_cf` and any hook.
* Force renew. (require `--ecc`)(I've not tried but auto renew should have same issue)
* The `example.com.conf` is broken. In my case, following configurations are disappeared:
  >Le_Domain='example.com'
  >Le_Alt='\*.example.com,\*.sub.example.com'
  >Le_Webroot='dns_cf'
  >Le_PreHook=''
  >Le_PostHook=''
  >Le_RenewHook='cat fullchain.cer example.com.key|tee combined.pem'
  >Le_DNSSleep='60'

Debug log
-----------------
Not available, I don't have spare domain to test.

Related issue
-----------------
#1172

**Comment by der-eismann** [comment]
Can confirm here... But it looks like everytime I use `--force` with `--issue` **OR** `--renew` a part of the config is deleted.
Before with 
`acme.sh --issue -d '*.example.com' --dns dns_example --renew-hook 'echo "Successful"'`:
```
Le_API='https://acme-v02.api.letsencrypt.org/directory'
Le_Keylength=''
Le_OrderFinalize='https://acme-v02.api.letsencrypt.org/acme/finalize/34615728/4999429'
Le_LinkCert='https://acme-v02.api.letsencrypt.org/acme/cert/xyz'
Le_CertCreateTime='1526040000'
Le_CertCreateTimeStr='Fri May 11 12:00:00 UTC 2018'
Le_NextRenewTimeStr='Tue Jul 10 12:00:00 UTC 2018'
Le_NextRenewTime='1531137000'
Le_Domain='*.example.com'
Le_Alt='no'
Le_Webroot='dns_example'
Le_PreHook=''
Le_PostHook=''
Le_RenewHook='echo "Successful"'
```

After `acme.sh --renew --force -d *.example.com`:
```
Le_API='https://acme-v02.api.letsencrypt.org/directory'
Le_Keylength=''
Le_OrderFinalize='https://acme-v02.api.letsencrypt.org/acme/finalize/34615728/4999671'
Le_LinkCert='https://acme-v02.api.letsencrypt.org/acme/cert/xyz'
Le_CertCreateTime='1526040000'
Le_CertCreateTimeStr='Fri May 11 12:03:00 UTC 2018'
Le_NextRenewTimeStr='Tue Jul 10 12:03:00 UTC 2018'
Le_NextRenewTime='1531137800'
```

**Comment by der-eismann** [comment]
Looks like the reason is the renew hook, more precisely the pipe inside it... did some testing and this was the result:
```
+ sed 's|^Le_RenewHook=.*$|Le_RenewHook='\''echo "Certificate for \"*.example.com\" was renewed" | mailx -s "Certificate for \"*.example.com\" was renewed" production@example.com'\''|'
sed: -e expression #1, char 111: unknown option to `s'
```
I think the problem is that `sed` takes the pipe of mailx as it's end, doesn't know what to do and as a result the script can't read the rest of the config.
@Neilpang sorry to bother you, but since this was the way you recommended using the renew hook [here](https://github.com/Neilpang/acme.sh/issues/729#issuecomment-287021611), do you have any alternative or workaround?

**Comment by frkca** [comment]
Fixed in #1969 - just for those searching for it :-) Thanks @Neilpang !
